src/libs/actions/Task.js

```diff
@@ -414,7 +431,17 @@ function editTaskAndNavigate(report, ownerAccountID, {title, description, assign
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`,
-            value: {reportName: report.reportName, description: report.description, assignee: report.managerEmail, assigneeAccountID: report.managerID},
+            value: {
+                reportName: report.reportName,
+                description: report.description,
+                assignee: report.managerEmail,
+                assigneeAccountID: report.managerID,
+                pendingFields: {
+                    ...(title && {reportName: null}),
+                    ...(description && {description: null}),
+                    ...(assigneeAccountID && {managerID: null}),
+                },
+            },
```

Comment:
I think it's better to show the user a RBR error message when editing fails, and to dismiss the pending fields when the error message is dismissed.

Let's remove this and implement that.

Reply:
Nice thanks. I see that we're reverting the edited fields here too. I though we didn't want to do that, but I was wrong. `editReportComment` does the same so all good 👍